extensions/wikia/CommunityPage/CommunityPage.i18n.php

```diff
@@ -63,14 +63,14 @@
 	'communitypage-no-members' => 'This community has no members',
 	'communitypage-help-us-grow' => "Help us grow '''{{SITENAME}}'''!",
 	'communitypage-new' => 'New',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "communitypage-new" 13 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)